src/app.py

```diff
@@ -12,13 +13,12 @@
 def say_hello_world():
     return {'result': "Flask says Hello World"}
 
-@app.route('/surgeo', methods=["GET"])
+@app.route('/surgeo', methods=["GET", "POST"])
```

Comment:
Just curious, why did you add Post here? I actually think Post is not a bad idea for this endpoint (there are a few differences btwn get and post). We can table this discussion to another time, good to note.